packages/integration-objects/test/ValidatePattern.test.js

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+const { validatePattern } = require('../srv/validations/ValidatePattern');
+const Constants = require('../srv/utils/Constants');
+
+describe('validate pattern', function () {
+  it('validatePattern method does not return any error', function () {
```

Comment:
Minor: I have seen this in many places - technically there is a difference between throwing and returning an error. What you want to check here is that it is thrown (or not thrown) and not returned.

Reply:
Fixed